test/sqllogictest/transform/lifting.slt

```diff
@@ -864,13 +872,16 @@ query T multiline
 explain with(arity, join_impls) select * from t1 as a1 join t1 as a2 on (a2.f2 = (select 6 from t1)) where a2.f2 = 9;
 ----
 Explained Query:
-  CrossJoin type=differential // { arity: 4 }
+  CrossJoin type=delta // { arity: 4 }
```

Comment:
Another apparently good thing to check!

Reply:
Same as above, I guess. (Didn't check it in detail.)